### ISSUE TITLE
Logout dialog lacks keyboard usability (not focused)

### ISSUE THREAD

**Issue by autarkper** [linuxmint/Cinnamon#710]
I'm running Cinnamon/Ubuntu 12.04,  (version 1.4.0-UP3-0ubuntu1~precise1), and I find that the Logout dialog (activated by CTRL+ALT+DEL) does not have initial keyboard focus, nor am I able to focus it using ALT+TAB, nor does it appear in the window tray, and my keyboard input goes to whatever had keyboard focus before bringing the dialog up. The user is thus forced to use the mouse to interact with the dialog, and if they try to use the keyboard they might accidentally invoke some unintended behavior in the application they were working with.

The same goes for the Shutdown dialog, invoked by pressing the computer's Power button.
